app/javascript/containers/PlanetsIndexContainer.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState, useEffect} from 'react'
+import App from '../react/components/App'
+import PlanetTiles from './PlanetTiles'
+
+const PlanetsIndexContainer = props => {
+
+  let [planets, setPlanets] = useState([])
+
+  useEffect(() => {
+    fetch('https://swapi.co/api/planets/')
```

Comment:
you have a trailing `/` here, which is probably why you are hitting your show endpoint and not your index. remove the trailing slash